docs/docs/dev_docs/updating.md

```diff
@@ -2,103 +2,40 @@
 title: Updating
 ---
 
-## TL;DR
-
-1. **Updating the sandbox:**
-
-- If you installed sandbox via docker, run:
+:::info
 
-```shell
-/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
-```
+The `@aztec/aztec-sandbox` and `@aztec/cli` packages published to npm are deprecated in favor of Docker. If you've installed the sandbox or the CLI via NPM, uninstall them and remove them from your project dependencies and install via Docker.
 
-- If you have installed via an npm package then step 3 handles the update.
+:::
 
-2. **Updating Aztec-CLI:**
+## TL;DR
 
-- The above command also downloads the aztec-cli if a node package version of the CLI isn't found locally.
-- If you have globally installed the CLI previously, then run:
+1. Updating the sandbox and CLI:
 
 ```shell
-npm install -g @aztec/cli
+/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
```

Comment:
I think this will eventually get changed to install.aztec.network but I'm not sure when.

Reply:
Cool, ill leave it for now then.